app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for human in people:
+        person_list.append(Person(name=human["name"], age=human["age"]))
+
+    for person in people:
```

Comment:
To simplify code extract person before checking:

`current_person = Person.people[person["name"]]`